cmd/zoekt-sourcegraph-indexserver/index_test.go

```diff
@@ -687,6 +616,10 @@ func TestIndex(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			var got []string
 			runCmd := func(c *exec.Cmd) error {
+				if c.Env != nil {
+					t.Fatal("expected nil Env for command. Tenant Enforcement relies on us inheritting the parent process environment.")
+				}
+
```

Comment:
How this motivated?

Reply:
It's future proofing this code. c.Env == nil means we inherit the parent process environment. IE we will inherit tenant enforcement/etc environment variables which we now rely on being inheritted.